google/cloud/testing_util/example_driver.cc

```diff
@@ -85,9 +89,16 @@ int Example::Run(int argc, char const* const argv[]) try {
 void Example::PrintUsage(std::string const& cmd, std::string const& msg) {
   auto last_slash = cmd.find_last_of('/');
   auto program = cmd.substr(last_slash + 1);
+  if(msg.length() != 0) {
```

Comment:
It would be better to factor out the common parts of the if/else branches.  It also doesn't look like we need an extra newline after `full_usage_`.  And, prefer `!empty()` over `length() != 0`.  So, ...
```
  if (!msg.empty()) std::cerr << msg << "\n";
  std::cerr << "Usage: " << program << " <command> [arguments]\n\n"
            << "Commands:\n" << full_usage_;
```